specification/mediaservices/resource-manager/Microsoft.Media/stable/2021-11-01/AssetsAndAssetFilters.json

```diff
@@ -0,0 +1,2032 @@
+            }
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/MediaServicesAccountName"
+          },
+          {
+            "name": "assetName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The Asset name."
+          },
+          {
+            "name": "filterName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The Asset Filter name"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          }
+        ],
+        "x-ms-examples": {
+          "Delete an Asset Filter": {
+            "$ref": "examples/assetFilters-delete.json"
+          }
+        }
+      },
+      "patch": {
+        "summary": "Update an Asset Filter",
+        "description": "Updates an existing Asset Filter associated with the specified Asset.",
+        "operationId": "AssetFilters_Update",
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/AssetFilter"
+            }
+          },
+          "default": {
+            "description": "Detailed error information.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/MediaServicesAccountName"
+          },
+          {
+            "name": "assetName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The Asset name."
+          },
+          {
+            "name": "filterName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The Asset Filter name"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/AssetFilter"
+            },
+            "description": "The request parameters"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          }
+        ],
+        "x-ms-examples": {
+          "Update an Asset Filter": {
+            "$ref": "examples/assetFilters-update.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Media/mediaServices/{accountName}/assets/{assetName}/tracks": {
```

Comment:
this should be marked as pageable (https://armwiki.azurewebsites.net/api_contracts/guidelines/openapi.html#oapi009-always-add-x-ms-pageable-to-list-calls)